pkg/snapshot/README.md

```diff
@@ -22,76 +21,74 @@ Please use with caution.
 
 ## Prerequisites
 
-1. Docker or Tyk Operator binaries,
+1. Docker or Tyk Operator binaries
 2. Credentials to connect Tyk Dashboard or Gateway. Please visit [Tyk Docs](https://tyk.io/docs/tyk-stack/tyk-operator/installing-tyk-operator) for details.
 
-## Preparation
-1. Specify the Kubernetes resource names in `Config Data`
+## Preparation - API metadata
```

Comment:
Since the header is changed, the corresponding link to this header needs an update. Please update line 18 where this header is linked to `Preparation` header.